assets/src/stories-editor/plugins/story-settings-panel.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { metaToAttributeNames } from '../helpers';
+
+/**
+ * WordPress dependencies
+ */
+import { PluginDocumentSettingPanel } from '@wordpress/edit-post';
+import { __ } from '@wordpress/i18n';
+import { compose } from '@wordpress/compose';
+import {
+	withSelect,
+	withDispatch,
+} from '@wordpress/data';
+import {
+	SelectControl,
+	RangeControl,
+} from '@wordpress/components';
+
+const MetaFields = ( props ) => {
+	const {
+		meta,
+		autoAdvanceAfterOptions,
+		updateMeta,
+	} = props;
+
+	const {
+		autoAdvanceAfter,
+		autoAdvanceAfterDuration,
+	} = metaToAttributeNames( meta );
+
+	const currentOption = autoAdvanceAfterOptions.find( ( i ) => i.value === autoAdvanceAfter ) || {};
+
+	return (
+		<>
+			<p>{ __( 'These settings are applied to new pages.', 'amp' ) }</p>
```

Comment:
Why only new pages?

When I create a new story and immediately change this setting, I'd expect this to change the value for the existing first page as well.

Reply:
@swissspidy Because it seems more consistent than any alternative where changing a story-wide settings may or may not affect any of the existing pages without proper explanation.

Ideally, I believe, it should be possible to _see_ a link between page settings to story settings - and the UI should clearly show which page settings are linked to story settings and which are changed and applied to that particular page only.

We might consider creating a UX ticket to investigate the various options here and enhance the experience in a later iteration.

I'm ready to defer to your experience and implement a change, just please let me know when you think should story settings _stop_ immediately affecting the page settings - is it after one of the page settings is changed by the user _or_ when a new page is created?

Reply:
I agree with @swissspidy global settings should allow to everything, as the name implies. Other this is really confusing for users.

Reply:
@swissspidy Created a ticket to figure out which UI updates does this feature need and linked both your relevant comments on this PR: https://github.com/ampproject/amp-wp/issues/3238